src/app/effects/work-item.effects.ts

```diff
@@ -242,15 +242,13 @@ export class WorkItemEffects {
           );
           const staticPayload = this.workItemMapper.toServiceModel(wp.payload);
 
-          // We don't update work item type
-          // So we remove it from the payload
           payload = cleanObject({
             ...staticPayload,
             ...{ attributes: {
                  ...staticPayload.attributes,
                  ...dynamicPayload.attributes
             }}
-          }, ['baseType']);
+          });
```

Comment:
If you remove `'baseType'` then on dynamic field patch request `baseType` will be included in the payload. Please check.

Reply:
And if it stays then it won't work for type update. I don't see a problem in sending the baseType for dynamic field update.

Reply:
I'm not sure but I think It will generate an extra event for workItemType whenever a dynamic field update happens.

Reply:
It'll not create an event for the same type, confirmed by @jarifibrahim